doc/migrations/v0.46-v1.0.0.md

```diff
@@ -365,6 +366,52 @@ const node = await createLibp2p({
 })
 ```
 
+## Keeping the same PeerId after a restart
+
+The libp2p PeerId is a public/private keypair. libp2p uses the keychain to encrypt it at rest when the node is not running.
```

Comment:
```suggestion
Given the libp2p `PeerId` is a public/private keypair, it is encrypted by libp2p's `Keychain` whenever the node is not running.
```

Reply:
I can appreciate the familiarity with the terminology but at the same time I think " node not running" implies at rest which is why I wanted to remove the redundancy. 

```suggestion
Given the libp2p `PeerId` is a public/private keypair, it is encrypted at rest by libp2p's `Keychain`
```